task-requests/details/style.css

```diff
@@ -512,6 +515,18 @@ table tr td {
   white-space: normal;
 }
 
+.requestor_description_details img {
+  width: 100%;
+}
+
+.requestor_description_details table {
+  display: block;
+  width: 100%;
```

Comment:
Do we need 100%, isn't this going to be overridden by the next width styling mentioned?